### PR TITLE
Felix/movo2 launch fixes

### PR DESCRIPTION
**Pull request by FelixKHWong**
Movo2 launch file fixes. Will also need some changes to visudo to get the time-sync running but that must be done locally on every movo

### CODE REVIEW

File: movo_robot/movo_bringup/launch/movo2.launch [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	<!-- Time synchonization for sibot2 -->
 	<node pkg="si_utils" type="sync_chrony" 
 		args="10.66.171.2"
-		name="time_sync" output="screen"/>
+		name="time_sync" launch-prefix="sudo" output="screen"/>
```
Comment (younesr1) [inline movo_robot/movo_bringup/launch/movo2.launch:24]: nice!. so this asks us to enter the passwd when we launch ros2.launch right? and only uses root priviliged for the sync chrony node?

File: movo_7dof_moveit_config/launch/move_group.launch [hunk 2]
```diff
@@ -50,7 +50,7 @@
 
     <!-- MoveGroup capabilities to load -->
     <param name="capabilities" value="move_group/MoveGroupCartesianPathService
-				      move_group/MoveGroupExecuteService
+				      move_group/MoveGroupExecuteTrajectoryAction
```
Comment (younesr1) [inline movo_7dof_moveit_config/launch/move_group.launch:53]: is this a soln u found online?

**Review by younesr1 — APPROVED** [review]

**Review by le2tang — APPROVED** [review]

File: movo_7dof_moveit_config/launch/move_group.launch (hunk 2 above)
Comment (FelixKHWong) [reply movo_7dof_moveit_config/launch/move_group.launch:53]: Yeah I found that MoveGroupExecuteService is deprecated and replaced with "TrajectoryAction"
https://github.com/ros-industrial/universal_robot/issues/413 
https://github.com/ros-planning/moveit/issues/201

File: movo_robot/movo_bringup/launch/movo2.launch (hunk 1 above)
Comment (FelixKHWong) [reply movo_robot/movo_bringup/launch/movo2.launch:24]: By editing the visudo file with "sudo visudo", we are able to allow sudo to run the specific script without a password. This solution is elegant and also safe. See this PR for more details https://github.com/nightingale-project/kinova-movo/pull/6#issuecomment-1279790947

**Pull request merged**